angr/analyses/decompiler/structured_codegen.py

```diff
@@ -211,7 +220,6 @@ def c_repr_chunks(self, indent=0):
     def indent_str(indent=0):
         return " " * indent
 
-
```

Comment:
Why are you removing the blank line? PEP8 requires two blank lines between classes.

Reply:
fixed